package.json

```diff
@@ -7,6 +7,7 @@
     "test": "test"
   },
   "devDependencies": {
+    "core-assert": "^0.1.3",
```

Comment:
Oh that, sorry yeah I just did it from CLI with `npm install --save-dev` and let it pick the dependency. I'll update although in this case ~ or ^ will behave strictly equally AFAIK.